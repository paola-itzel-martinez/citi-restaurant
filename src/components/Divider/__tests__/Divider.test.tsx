import React from "react";
import { customRender } from "../../../../tests";
import { Divider } from "..";

describe("<Card />", () => {
  it("Should have a default css rules by default", () => {
    const { container } = customRender(<Divider />);

    expect(container).toMatchSnapshot();
  });
});
