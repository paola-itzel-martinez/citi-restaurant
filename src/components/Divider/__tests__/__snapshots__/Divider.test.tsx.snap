// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> Should have a default css rules by default 1`] = `
.c0 {
  border-color: #e0e6eb;
  border-style: solid;
  border-width: 0px 0px thin;
  -webkit-align-self: undefined;
  -ms-flex-item-align: undefined;
  align-self: undefined;
  height: undefined;
  width: 100%;
  margin-block: unset;
  margin-inline: unset;
}

<div>
  <hr
    class="c0"
  />
</div>
`;
